### PR TITLE
keyWindow should be touched from main thread in iOS

### PR DESCRIPTION
**Pull request by Jlexyc**
Added ticket subject configuration for iOS
Fixed main thread issue

### CODE REVIEW

File: ios/RNZenDeskSupport.m [hunk 1]
```diff
@@ -104,28 +102,34 @@ @implementation RNZenDeskSupport
     [self showLabelsWithOptions:labels options:nil];
 }
 
-RCT_EXPORT_METHOD(callSupport:(NSDictionary *)customFields) {
-    UIWindow *window=[UIApplication sharedApplication].keyWindow;
-    UIViewController *vc = [window rootViewController];
-
+RCT_EXPORT_METHOD(callSupport:(NSDictionary *)fields) {
+    NSDictionary *customFields = [RCTConvert NSDictionary:fields[@"customFields"]];
```
Comment (RobertSheaO) [inline ios/RNZenDeskSupport.m]: This looks great, thank you! Can you also update the README on how to use this method with the new changes?

Comment (Jlexyc) [reply ios/RNZenDeskSupport.m]: Sorry man, I did this changes after PR and forgot to change readme. Now it's done :)

**Comment by RobertSheaO** [comment]
I'll write an Android implementation for this tomorrow and merge in your changes. I don't want to segment the library by device.

**Comment by Jlexyc** [comment]
I just didn't implement (adopt) android app so can't test android changes. But in future i'll use for Android too. tnx.

**Comment by RobertSheaO** [comment]
Can you update the title to reflect the addition of the tags and subject. keyWindow has been implemented.

**Pull request closed without merging**